actionview/lib/action_view/buffers.rb

```diff
@@ -18,24 +18,56 @@ module ActionView
   #   sbuf << 5
   #   puts sbuf # => "hello\u0005"
   #
-  class OutputBuffer < ActiveSupport::SafeBuffer # :nodoc:
-    def initialize(*)
-      super
-      encode!
+  class OutputBuffer
+    def initialize(buffer = "")
+      @buffer = buffer.dup
+      @buffer.encode!
+    end
+
+    delegate :length, :inspect, :blank?, :force_encoding, :empty?, :ascii_only?, :html_safe, to: :@buffer
+    delegate_missing_to :@buffer # TODO: this is a shortcut
```

Comment:
We may not even want to implement all the string methods here. It's probably preferable to cast `OutputBuffer` instances back to `ActiveSupport::SafeBuffer` early.

Reply:
Ok, removed.